### PR TITLE
Print environment variables on startup

### PR DESCRIPTION
**Pull request by PabloCastellano**

### CODE REVIEW

File: apps/rest-api-server/api/env.ts [hunk 1]
```diff
@@ -6,7 +7,14 @@ export const env = createEnv({
     NODE_ENV: z.enum(["development", "test", "production"]).optional(),
     METRICS_ENABLED: booleanSchema.default("true"),
     TRACES_ENABLED: booleanSchema.default("false"),
+    PRINT_CONFIG: booleanSchema.default("false"),
```
Comment (PabloCastellano) [inline apps/rest-api-server/api/env.ts:10]: Since this is only shown once on startup I would remove this option

File: apps/rest-api-server/api/instrumentation.ts [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 
 import { setUpOpenTelemetry } from "@blobscan/open-telemetry";
 
 import { env } from "./env";
 
 if (env.TRACES_ENABLED) {
   setUpOpenTelemetry("blobscan_rest_api", {
-    instrumentations: [
-      new ExpressInstrumentation(),
-      new WinstonInstrumentation(),
```
Comment (PabloCastellano) [inline apps/rest-api-server/api/instrumentation.ts:12]: Why did you remove WinstonInstrumentation? Is it related to this PR?

File: packages/blob-storage-manager/src/env.ts [hunk 3]
```diff
@@ -12,9 +13,28 @@ export const env = createEnv({
     GOOGLE_STORAGE_ENABLED: booleanSchema.default("false"),
     POSTGRES_STORAGE_ENABLED: booleanSchema.default("true"),
     SWARM_STORAGE_ENABLED: booleanSchema.default("false"),
+    PRINT_CONFIG: booleanSchema.default("false"),
```
Comment (PabloCastellano) [inline packages/blob-storage-manager/src/env.ts:16]: Same, I don't see the point of having it

Comment (PJColombo) [reply packages/blob-storage-manager/src/env.ts:16]: Yeah you're right. I'll rollback this.
